engine/lib/deprecated-1.10.php

```diff
@@ -36,7 +36,9 @@ function file_get_general_file_type($mime_type) {
  */
 function elgg_get_access_object() {
 	elgg_deprecated_notice(__FUNCTION__ . ' is deprecated. Use elgg_get/set_ignore_access()', '1.10');
-	return _elgg_services()->access;
+
+	// note we've already broken this function (it used to return an ElggAccess)
```

Comment:
It was introduced in 1.7 and [marked private in 1.8](https://github.com/mrclay/Elgg-leaf/commit/cb5df9222c274a6fcf9263cc5fb905b7a76add40#diff-23234856e075522f9fb90e4cbc1d4ae1R906). I think I agree it can go.